tools/cldr-apps/js/src/views/GeneralInfo.vue

```diff
@@ -7,12 +7,12 @@
       @click="insertDashboard"
       class="cldr-nav-btn btn-primary open-dash"
       type="button"
-      v-if="!dashButtonShouldBeVisible()"
+      v-if="dashButtonShouldBeVisible()"
     >
       Open Dashboard
     </button>
 
-    <!-- <cldr-overall-errors /> -->
+    <cldr-overall-errors />
```

Comment:
wait, did you mean to re-enable this? Why? I think we need to discuss first!

Reply:
I saw it too just after the commit, sorry about the noise! It was temporary for testing the call to cldrClient that motivated the whole tangent